README.md

```diff
@@ -0,0 +1,41 @@
+# Cincinnati
+
+Cincinnati is a service to transform update information as modeled in a Docker registry (such as Quay.io) into a graph representation that can be consumed by update clients. It is the spiritual successor to the Omaha protocol used to udate Google Chrome and Google Chrome OS based operating systems.
```

Comment:
This isn't very accurate. I'm in favor of leaving off the descriptions for now. I'm in the (slow) process of writing a README which talks about the overall design a little more.

Reply:
The repo needs something. I suggest you start with what I have and iterate until it's as nice as you want it to be.

Reply:
What part of it isn't accurate?